scripts/nixpkgs.nix

```diff
@@ -0,0 +1,12 @@
+{ system ? builtins.currentSystem, ... }@args:
+
+(import
+  (
+    let lock = builtins.fromJSON (builtins.readFile ../flake.lock); in
+    fetchTarball {
+      url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+      sha256 = lock.nodes.flake-compat.locked.narHash;
+    }
+  )
+  { src = ../.; }
+).defaultNix.outputs.nixpkgs.${system}
```

Comment:
Note for later to refactor our flakes.

Reply:
Would you mind expanding on the thought?